File: Cpp/Math/Middle of Three.cpp
//https://www.geeksforgeeks.org/problems/middle-of-three2926/1
class Solution {
  public:
    int middle(int A, int B, int C) {
        // code here
        if (A>B && A>C){
            return B>C ? B:C;
        }
        else if (B>A && B>C){
            return A>C ? A:C;
        }
        else{
             return A>B ? A:B;
        }
    }
};

File: Python/Math/Middle of Three.py
//https://www.geeksforgeeks.org/problems/middle-of-three2926/1
class Solution:
    def middle(self, a, b, c):
        #code here
        if a > b and a > c:
            return b if b > c else c
        elif b > a and b > c:
            return a if a > c else c
        else:
            return a if a > b else b
